operators/src/source/gdal_source.rs

```diff
@@ -445,75 +287,65 @@ where
                     GridBoundingBox2D::new(tile_idx_ul, tile_idx_lr)?,
                 )?;
 
-                let mut tile_raster = Grid2D::new_filled(tile_grid, T::zero(), None);
+                let mut tile_raster = Grid2D::new_filled(output_shape, T::zero(), None);
                 tile_raster.grid_blit_from(dataset_raster)?;
                 tile_raster
             }
         };
 
-        Ok(RasterTile2D::new_with_tile_info(
-            time_interval,
-            tile_information,
-            result_raster,
-        ))
+        Ok(result_raster)
     }
 
     ///
     /// A stream of `RasterTile2D`
     ///
     pub fn tile_stream(
         &self,
-        bbox: BoundingBox2D,
-        time_interval: TimeInterval,
-        spatial_resolution: SpatialResolution,
+        query: QueryRectangle,
```

Comment:
if we use "QueryRectangle" also as STBoundingBox then we should rename it.

Reply:
I don't use it like this, I need the resolution here, too.